tsconfig.json

```diff
@@ -1,28 +1,14 @@
 {
     "compilerOptions": {
-        "target": "es6",
+        "baseUrl": "src",
+        "downlevelIteration": true,
+        "lib": ["dom", "es2015", "es2016", "es2017"],
+        "module": "es2015",
+        "moduleResolution": "node",
+        "pretty": true,
         "strict": true,
-        "module": "none",
-        "sourceMap": true,
-        "outDir": "docs",
-        "outFile": "docs/gtr-cof.js",
         "types": []
     },
-    "files": [
-        "src/mod-module.ts",
-        "src/events-module.ts",
-        "src/cookie-module.ts",
-        "src/music-module.ts",
-        "src/state-module.ts",
-        "src/cof-module.ts",
-        "src/tonics-module.ts",
-        "src/chord-interval-module.ts",
-        "src/modes-module.ts",
-        "src/gtr-module.ts",
-        "src/tuning-module.ts",
-        "src/settings-module.ts",
-        "src/scale-family-module.ts",
-        "src/midi-module.ts",
-        "src/gtr-cof.ts"
-    ]
-}
\ No newline at end of file
+    "include": ["typings/*.d.ts", "src/**/*.ts"],
```

Comment:
For some pesky Type errors I added a `typings` folder for dealing with webmidi types. I'm sure this can be refactored out but for now was added to satisfy the type checker.